test/common/crudtester.go

```diff
@@ -714,27 +715,133 @@ func (c *FederatedTypeCrudTester) versionForCluster(version *fedv1a1.PropagatedV
 	return ""
 }
 
+func (c *FederatedTypeCrudTester) CheckRemoteStatus(fedObject *unstructured.Unstructured, targetObject *unstructured.Unstructured) {
+	for clusterName := range c.testClusters {
+		clusterConfig := c.testClusters[clusterName].Config
+
+		kubeClient := kubeclientset.NewForConfigOrDie(clusterConfig)
+		WaitForNamespaceOrDie(c.tl, kubeClient, clusterName, targetObject.GetNamespace(),
+			c.waitInterval, 30*time.Second)
+
+		util.AddManagedLabel(targetObject)
+		labeledObj, err := CreateResource(clusterConfig, c.typeConfig.GetTargetType(), targetObject)
+		if err != nil {
+			c.tl.Fatalf("Failed to create labeled resource in cluster %q: %v", clusterName, err)
+		}
+
+		clusterClient := genericclient.NewForConfigOrDie(clusterConfig)
+		defer func() {
+			err := clusterClient.Delete(context.TODO(), labeledObj, labeledObj.GetNamespace(), labeledObj.GetName())
+			if err != nil {
+				c.tl.Fatalf("Unexpected error: %v", err)
+			}
+		}()
+
+		c.tl.Log("Checking that the resource has status")
+		var objStatus interface{}
+		err = wait.PollImmediate(c.waitInterval, wait.ForeverTestTimeout, func() (bool, error) {
+			obj := &unstructured.Unstructured{}
+			obj.SetGroupVersionKind(labeledObj.GroupVersionKind())
+			err := clusterClient.Get(context.TODO(), obj, labeledObj.GetNamespace(), labeledObj.GetName())
+			if err != nil {
+				c.tl.Errorf("Error retrieving kubefed cluster object resource: %v", err)
+				return false, nil
+			}
+
+			objStatus = obj.Object[util.StatusField]
+			c.tl.Logf("Show kubefed cluster object status: %v", objStatus)
+			return (obj.Object[util.StatusField] != nil), nil
```

Comment:
```suggestion
			return obj.Object[util.StatusField] != nil, nil
```
parens are not needed